agenta-web/src/components/pages/overview/automaticEvaluation/AutomaticEvalOverview.tsx

```diff
@@ -253,7 +253,7 @@ const AutomaticEvalOverview = () => {
                                 <Popover
                                     key={index}
                                     placement="bottom"
-                                    trigger={"click"}
+                                    trigger={"hover"}
```

Comment:
@ashrafchowdury why this change?

Reply:
Now, to see the Popover component, we need to hover on Automatic evaluation result items instead of clicking on them as per the design.